banzai/tic-tac-toe.rb

```diff
@@ -0,0 +1,108 @@
+# this gem injects a color method to the string when it is displayed by console
+require 'colorize' 
+
+@level = 2
```

Comment:
Is there a reason why this is an instance variable? (with `@`)
Instance variables are intended to be used inside a Class, in your case a normal local variable `(starting with a letter or underscore) could have been a better option.